pyvlx/parameter.py

```diff
@@ -7,6 +7,7 @@ class Parameter():
 
     UNKNOWN_VALUE = 63487  # F7 FF
     CURRENT_POSITION = 53760  # D2 00
+    CURRENT_INTENSITY = 53760  # D2 00
     MAX = 51200  # C8 00
     MIN = 0  # 00 00
     ON = 0  # 00 00
```

Comment:
I think adding a `  #pylint disable=invalid-name` should be added here. "ON" is exactly the right name.

Reply:
Sorry, I do not understand.

Reply:
[Pylint](https://www.pylint.org/) is complaining, that "ON" is not a correct name for a variable. I wanted to say, that we should ignore this warning.